src/templates/index.ejs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" prefix="og: http://ogp.me/ns#">
+  <head>
+    <%- include _head %>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self' www.google-analytics.com *.infura.io 'sha256-87Klt/UgO2rT2F3T3VzTH5GP8hORFujTILA05Z7dcQw='; connect-src *.infura.io storage.thousandetherhomepage.com http://127.0.0.1:8545/rpc/; style-src 'self' 'unsafe-inline'; img-src * data:;" />
```

Comment:
I left these here because I didn't want to update CSP to include whatever we may use in press & faq page (i.e. youtube embed js), and I wasn't sure if we should bother with separate meta tags per page.

I'm happy leaving as is for now unless you think either is important.

Reply:
Yea makes sense, I think the vuejs app entrypoint is the only place we superneed CSP for now.